src/main/java/com/android/volley/cronet/CronetHttpStack.java

```diff
@@ -70,7 +64,7 @@ public void executeRequest(
             final Request<?> request,
             final Map<String, String> additionalHeaders,
             final OnRequestComplete callback) {
-        if (mBlockingExecutor == null || mCallbackExecutor == null) {
+        if (getBlockingExecutor() == null || getNonBlockingExecutor() == null) {
             throw new IllegalStateException("Must set blocking / non-blocking executors");
```

Comment:
super nit, but I think this should be an "and" ("Must set blocking and non-blocking executors")

Reply:
Shouldn't it be that if either of them are null, we throw the exception?

Reply:
Yep, I'm just referring to the IllegalStateException method, since both should be set. But yes, the code itself is right (should throw if either are null since both need to be set)

Reply:
*the IllegalStateException message